ports/espressif/boards/heltec_esp32s3_wifi_lora_v3/sdkconfig

```diff
@@ -0,0 +1,16 @@
+#
+# Espressif IoT Development Framework Configuration
+#
+#
+# Component config
+#
+#
+# LWIP
+#
+CONFIG_ESP_PHY_ENABLE_USB=y
```

Comment:
I doubt you need this if the native USB isn't connected.

Reply:
Hmm, probably not.  I thought I enabled it for a reason but I can't remember why.

I'll try disabling it, building, and flashing.

If it works fine I'll remove it.  If it doesn't, I'll add a comment describing what reason it should be enabled.

Reply:
Loads just fine, not sure where I thought I needed it.  Removing!